lib/spoom/cli/coverage.rb

```diff
@@ -15,7 +15,7 @@ class Coverage < Thor
 
       desc "snapshot", "Run srb tc and display metrics"
       option :save, type: :string, lazy_default: DATA_DIR, desc: "Save snapshot data as json"
-      option :rbi, type: :boolean, default: true, desc: "Exclude RBI files from metrics"
+      option :rbi, type: :boolean, default: true, desc: "Include RBI files in metrics"
```

Comment:
This is the other way around. It comes from a refactor and I forgot to change the description.